### ISSUE TITLE
Duplicate bug

### ISSUE THREAD

**Issue by Pascal-fe** [lazybearcreations/sketch-lazyboards#4]
Hey,

When I select a layer/folder and duplicate it to all artboards or to selected artboards a bug appears. The layer/folder is properly duplicated to the other artboards but it's also duplicated within the original artboard. 
![Duplication bug](https://user-images.githubusercontent.com/23357991/55481131-0092ac80-5622-11e9-8f9b-4ca74c31326d.png)

**Comment by Niknam** [comment]
I'm experiencing the same thing. Hope there is a fix planned for this. It's really a time-saving feature!

**Comment by sirme87** [comment]
Yes, I run into the same bug. If you delete the duplicates in the original document, all duplicates in other artboards will be removed also.
